modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1458,12 +1458,45 @@ public void scrollTo(int index) {
         if (cell != null) {
             scrollTo(cell);
         } else {
+            // see JDK-8197536
+            if (tryScrollOneCell(index, true))
+                return;
```

Comment:
Please enclose this in curly braces; our Java code conventions do not allow for single-statement targets of a conditional expression (unless on the same line as the conditional, which isn't desirable here).

Reply:
Done. Thanks.